react/index.tsx

```diff
@@ -58,23 +67,32 @@ const CategoryMenu = ({
       <div className={`${styles.sidebarContainer} ${styles.mobile}`}>
         <SideBar
           visible={sideBarVisible}
-          title={intl.formatMessage({
-            id: 'store/category-menu.departments.title',
-          })}
+          // title={intl.formatMessage({
+          //   id: 'store/category-menu.departments.title',
+          // })}
           departments={visibleDepartments}
           onClose={handleSidebarToggle}
           showSubcategories={showSubcategories}
+          mobileDisplayType={mobileDisplayType}
         />
-        <div className="flex pa4 pointer" onClick={handleSidebarToggle}>
+        <div
+          className="flex pa4 pointer items-center"
+          onClick={handleSidebarToggle}
+        >
           <IconMenu size={20} />
+          <p className="flex ml4">
+            {intl.formatMessage({
+              id: 'store/category-menu.departments.title',
+            })}
+          </p>
```

Comment:
It seems to be a breaking change, right? @victorhmp @icazevedo, can you help here?